compiler/qsc_rca/src/scaffolding.rs

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use crate::{
+    common::GlobalSpecId, ApplicationGeneratorSet, CallableComputeProperties,
+    ComputePropertiesLookup,
+};
+use qsc_data_structures::index_map::IndexMap;
+use qsc_fir::{
+    fir::{
+        self, BlockId, ExprId, LocalItemId, PackageId, StmtId, StoreBlockId, StoreExprId,
+        StoreItemId, StoreStmtId,
+    },
+    ty::FunctorSetValue,
+};
+
+/// Scaffolding used to build the package store compute properties.
+#[derive(Debug)]
+pub struct PackageStoreComputeProperties(IndexMap<PackageId, PackageComputeProperties>);
```

Comment:
On the topic of naming, I think using the same name for the scaffolding class that is internal and the public facing class can be a bit confusing, even though they are qualified by the module within the crate. Maybe something like `InternalStoreComputeProperies`?

Reply:
Renamed!